tempto-core/src/main/java/com/teradata/tempto/internal/hadoop/hdfs/WebHdfsClient.java

```diff
@@ -183,9 +183,7 @@ public void loadFile(String path, OutputStream outputStream)
         }
     }
 
-    @Override
-    @SuppressWarnings("unchecked")
-    public long getLength(String path)
+    private Object getAttributeValue(String path, String attribute)

```

Comment:
Why not make this public for the benefit of anybody who knows what they're doing and needs to query something that doesn't have a specific method exposed yet?